locales/es.po

```diff
@@ -0,0 +1,252 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: app 1.0\n"
+"mime-version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"x-generator: i18next-auto-keys CLI\n"
+"Language: es-ES\n"
+"PO-Revision-Date: 2025-11-06T13:58:13.061Z\n"
+"X-Generator: Autopo v1.0.5\n"
+
+#: src/renderer/pages/settings/create.messages.ts:10:19
+msgid "Advanced"
+msgstr "Avanzado"
+
+#: src/renderer/pages/settings/advanced_app_settings/language_selector/language_selector.messages.ts:2:22
+msgid "App Language"
+msgstr "Idioma de la Aplicación"
+
+#: src/renderer/pages/settings/create.messages.ts:9:22
+msgid "App Settings"
+msgstr "Configuración de la Aplicación"
+
+#: src/renderer/pages/settings/advanced_app_settings/advanced_app_settings.messages.ts:3:39
+msgid "Automatically install Slippi Launcher updates when they become available."
+msgstr ""
+"Instalar automáticamente las actualizaciones de Slippi Launcher cuando "
+"estén disponibles."
+
+#: src/renderer/pages/settings/create.messages.ts:5:15
+msgid "Chat"
+msgstr "Chat"
+
+#: src/renderer/app/header/header.messages.ts:3:26
+msgid "Check for updates"
+msgstr "Buscar actualizaciones"
+
+#: src/renderer/app/header/header.messages.ts:4:29
+msgid "Checking for updates..."
+msgstr "Buscando actualizaciones..."
+
+#: src/renderer/pages/settings/game_settings/game_settings.messages.ts:8:38
+msgid "Choose what happens when the Play button is pressed."
+msgstr "Elige qué sucede cuando se presiona el botón Jugar."
+
+#: src/renderer/pages/settings/advanced_app_settings/advanced_app_settings.messages.ts:4:25
+msgid "Clear Temporary Files"
+msgstr "Limpiar Archivos Temporales"
+
+#: src/renderer/pages/settings/advanced_app_settings/advanced_app_settings.messages.ts:7:21
+msgid "Clear files"
+msgstr "Limpiar archivos"
+
+#: src/renderer/pages/settings/game_settings/game_settings.messages.ts:4:31
+msgid "Close Dolphin to change this setting"
+msgstr "Cierra Dolphin para cambiar esta configuración"
+
+#: src/renderer/app/create.messages.ts:5:18
+msgid "Console Mirror"
+msgstr "Espejo de Consola"
```

Comment:
I think that Console Mirror should not be translated here, at least not the mirror word. I've never heard the expression "Espejo de consola" ever. I'd suggest against translating the term at all and use `"Mirror de Consola"` or use another term like console transmission/syncronization or maybe console cloning/duplication, tho these last two sound like you are cloning the console itself lol

Those would be:
- `"Transmisión de Consola"`
- `"Sincronización de Consola"`
- `"Duplicación de Consola"`
- `"Clonación de Consola"`

Reply:
Went with the first suggestion